### ISSUE TITLE
Workshop mods not working

### ISSUE THREAD

**Issue by therimmer96** [SEServerExtender/SEServerExtender#68]
When ever I add workshop mods to the sandbox.sbc file, SESE renames the file and creates a new one without the mods before starting the server, meaning that I don't get my mods. 

Is this a config option somewhere that I can change?

**Comment by johskar** [comment]
Tried adding them to the Mods field in SESE GUI server properties?

**Comment by therimmer96** [comment]
I am using the mod with no gui, so I can't

**Comment by dodexahedron** [comment]
In your SpaceEngineers-Dedicated.cfg file, which is located in your world save folder, make sure there is a LoadWorld tag and that it contains the path of your specific world save.

Example:
`<LoadWorld>C:\ProgramData\SpaceEngineersDedicated\MyWorld\Saves\MyWorld</LoadWorld>`

**Comment by therimmer96** [comment]
That was already done. Its renaming the sandbox.sbc  file INSIDE that world.

**Comment by dodexahedron** [comment]
SESE doesn't do that.
It doesn't directly modify any files inside your world.  Only the game and one of the functions in Essentials do that.  And Essentials doesn't rename anything, either - it just modifies Sandbox.sbc.

Can you pastebin your SpaceEngineers-Dedicated.cfg file as well as the Settings node from your Sandbox.sbc file?

**Comment by therimmer96** [comment]
Sorry for delay

Here, have a zip with the log, the config, and 2 versions of the world. 1 from after running SESE, and a copy of the world from before running, with the mods intact.

http://puu.sh/iBlNP/9e1d8f2d1e.zip

**Comment by Ragonz2** [comment]
Is there any update on this. We have the exact same issue and I even posted about it when Tyrsis was doing SESE - https://github.com/Tyrsis/SE-Community-Mod-API/issues/29

Its not a steam issue as otherwise loading in the mods from the workshop without using sese would fail. The issue only occurs when sese is used.

**Comment by Mattheus80** [comment]
habe you tried to delete the mods folder so SE can download them again?

**Comment by Ragonz2** [comment]
Yes. I have tried every conceivable way of getting it to work but it just will not.

Fresh server started in normal way without mods then stopped, mods added and started with SESE
Fresh server started with SESE without mods then mods added
Fresh server started with SESE without mods, then stopped then mods added, loaded with normal way then stopped then loaded with SESE
Existing server stopped then started with SESE

etc etc

**Comment by therimmer96** [comment]
If someone has a working server setup with Workshop mods and SESE, could they Zip it and send it so  I can compare it to my attempts?

**Comment by Mattheus80** [comment]
i'm confuse. sese is running without mods and if YouTube add mods sese stop running, right? i had the same Problem a few days ago, it was just one mod that causes the Problem. try to add a mod start sese and if it's running add next mod and so on. hope i could help.

**Comment by therimmer96** [comment]
I have been trying this for months, with several different mods. Even ones that just add basic blocks are causing the issue.

**Comment by Mattheus80** [comment]
sorry autocorrection, not youtube , you.

**Comment by therimmer96** [comment]
Has anyone had any luck with this yet?

**Comment by Ragonz2** [comment]
Any update? Tried it again with the latest update, Start server with mods specified in the Sandbox.sbc file of a world normally all works fine

Start the same world with SESE, all mods removed from sandbox.sbc file
